marinerescue-frontend/src/pages/ParentsPage.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import { Link } from "react-router-dom";
+
+import './AboutPage.scss';
+
+
+class ParentsPage extends React.Component {
+    render() {
+        return (
+            <main>
+                <div class="navbar">
+                    <Link to= "/">
+                    <div class="navbar-logo">
+                        <img src="images/logo.png"></img>
+                    </div>
+                    </Link>
+                    <Link to="/leaderboard">Leaderboard</Link>
+                    <Link to="/about">About</Link>
+                    <Link to="/parents">Parents</Link>
+                    <a href="#playnow"><button id="nav-bar-button">Play Now</button></a>
+                    <a href="profile">Profile</a>
+                    {/* <div class="navbar-logo">
+                        <img src="images/profile-logo.png"></img>
+                    </div> */}
+                </div>
```

Comment:
Can you please move this to be its own component <Navbar> so we can re-use it everyone on our website? Thanks!